uadapy/plotting/plots1D.py

```diff
@@ -184,15 +187,16 @@ def plot_1d_distribution(distributions, num_samples, plot_types:list, seed=55, f
                                    showmeans=True, meanline=True, meanprops=dict(color="black", linestyle='-'),
                                    medianprops=dict(linewidth=0), vert=kwargs.get('vert', True))
                     if 'violinplot' in plot_types:
-                        parts =  ax.violinplot(sample[:,index], positions=[k], showmeans=True, vert=kwargs.get('vert', True))
+                        parts =  ax.violinplot(sample[:,index], positions=[k], showmeans=kwargs.get('showmeans', False), vert=kwargs.get('vert', True))
                         for pc in parts['bodies']:
                             pc.set_facecolor(palette[k % len(palette)])
                             pc.set_edgecolor(palette[k % len(palette)])
                             pc.set_alpha(0.5)
                         parts['cbars'].remove()
                         parts['cmaxes'].remove()
                         parts['cmins'].remove()
-                        parts['cmeans'].set_edgecolor('black')
+                        if kwargs.get('showmeans', False):
+                          parts['cmeans'].set_edgecolor('black')
```

Comment:
would it not need to be the other case? if showmeans true -> use black edge color

Reply:
The "use black edge color" code is only executed if the **showmeans** keyword has been passed into the function as part of kwargs, and is set to true.
The **False** in the second slot of the kwargs.get function is a fallback value to use if the keyword doesn't exist, such as when it hasn't been specified by the user.
I've stuck to the way the **vert** keyword is handled in this function, however, **showmeans** could be made its own parameter, improving usability and readability in both the code and the function's signature.